packages/grafana-ui/src/components/PageLayout/PageToolbar.tsx

```diff
@@ -56,6 +56,7 @@ export const PageToolbar: FC<Props> = React.memo(
     const mainStyle = cx(
       'page-toolbar',
       styles.toolbar,
+      { [styles.noPageIcon]: !pageIcon },
```

Comment:
You can move this into the existing object, you don't need to pass a class name per object

Reply:
That is the line below here just add it to that

Reply:
yeah, you're right. my bad